### ISSUE TITLE
Portals only show when there is enough content in file

### ISSUE THREAD

**Issue by ghostbuster91** [cbochs/portal.nvim#43]
Hi, 

Thanks for the great plugin, really neat idea. 

I was testing my own custom query when I noticed that sometimes only one result is displayed. However, after a closer look it turned out that the other results are displayed as well but beneath the one that I was seeing (I discovered that by coincidence because I hit a label that was not connected to the visible portal but it still got me traveled). 

Soon, I realized that portals are only shown on a place that was occupied by some text in a buffer. 
In other words, if you try to open multiple portals in a file that has only one line you will see only one portal.

**Comment by cbochs** [comment]
Ah, this is a good catch. The windows are all rendered, but stacked on top of each other. Some incorrect assumptions that the window height and buffer height will always be the same (which is untrue with the example you pointed out).
